pkg/pureutils/purefa.go

```diff
@@ -3,6 +3,7 @@ package pureutils
 import (
 	"fmt"
 	"github.com/devans10/pugo/flasharray"
+	newflasharray "github.com/portworx/torpedo/drivers/pure/flasharray"
```

Comment:
[nit] can we do `tpflasharray` instead? "new" doesn't say much

Reply:
Changed